docu/files/SkeletonCreatorBuilderCLI.html.md

```diff
@@ -1,45 +1,65 @@
 
 Here is an example of a UI with its corresponding code preview:<br>
 
-  <img src="img/skeletor_ui_completed.png" class="img-click-modal" alt="The Skeleton Creator UI"/><br><br>
-  <img src="img/skeletor_code_sample.png" class="img-click-modal" alt="Code Preview"/><br><br>
+<img style="width: 30%;" max-width: 326px src="img/skeletor_ui_completed.png" class="img-click-modal" alt="The Skeleton Creator UI"/><br><br>
+<img style="width: 30%;" max-width: 326px src="img/skeletor_code_sample.png" class="img-click-modal" alt="Code Preview"/><br><br>
 
-  If you are using Chrome or Firefox, a download button is available in preview mode allowing you to download the code to a file.
+If you are using Chrome or Firefox, a download button is available in preview mode allowing you to download the code to a file.
 
-  <img src="img/skeletor_menu_download_button.png"  width="35%" class="img-click-modal" alt="The Skeleton Creator download button"/><br><br>
-  Internext Explorer and Safari do not support the download button. Instead, you can copy the HTML code directly from the broswer. <br><br>
+<img style="width: 30%;" max-width: 326px src="img/skeletor_menu_download_button.png"  width="35%" class="img-click-modal" alt="The Skeleton Creator download button"/><br><br>
+Internext Explorer and Safari do not support the download button. Instead, you can copy the HTML code directly from the broswer. <br><br>
 
 
-<h3>Component settings</h3>
+**Component settings**
 
 You can adjust the settings for each component in the Skeleton Creator. This allows you to assign custom text and and labeling to the component. For example, the <strong>Page Header</strong> component features a <strong>Back</strong> button, a long page title, as well as a <strong>Save</strong> and <strong>Cancel</strong> button by default. You can set the default text of the title and rename each of the buttons.
 
 <br>
-  <img src="img/component_settings.png" class="img-click-modal" alt="Code Preview"/>
+  <img style="width: 30%;" max-width: 326px src="img/component_settings.png" class="img-click-modal" alt="Code Preview"/><br><br>
+  
+
+<h3>Dev Mode to preview your module</h3>
+
+Now that you have created your builder module, you can enable Developer Mode. After enabling this mode, link the Builder to your local module that you created in the last section. This connection makes it possible to begin previewing the HMTL and adding navigation nodes to your module directly in the Builder. You cannot yet call APIs.<br>
+
+**Enabling Developer Mode**
+
+To enable Developer Mode take the following steps:
+
+1. Navigate to the project home page and click on Development. 
+2. Switch Developer Mode on. A dark gray action bar appears at the bottom of the screen from which you can access <b>Settings</b> or exit Developer Mode.
+3. Click on <b>Settings</b> on the action bar.
+4. To show the locally developed Builder module, switch on the toggle <b>Show local module</b>
```

Comment:
4. To show the locally-developed Builder module, switch on the toggle <b>Show local module</b>.